### ISSUE TITLE
Recommend to restructure the repo

### ISSUE THREAD

**Issue by leonardhyu** [xenanetworks/open-automation-test-suites#7]

**Comment by ArtemConstantinov** [comment]
No, use a `dev` brush or `pre_release` which can be a mirror of `main` where you can apply changes over the code

**Comment by fpfeng** [comment]
> 

yes, I mean we all working on main branch with pulll request instead of each of us have their own branch, after that Leo would not need to manual merge.

**Comment by ArtemConstantinov** [comment]
No works directly on the main branch. You can create another gathering brunch and everyone will commit in to it...
